rules/store-your-secrets-securely/rule.md

```diff
@@ -124,6 +124,22 @@ Figure: Good practice -  Overall rating: 8/10
 
 ::: greybox
 
+#### Use Dot Net User Secrets
+
+Pros:
+* Simple secret management for development environments
```

Comment:
for **local** development environments

Reply:
The reason I used **development** is because that is what the microsoft documentation uses here: https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows#secret-manager

Are we sure we want it to say local on our rule?